content/projects/blueocean.adoc

```diff
@@ -54,6 +54,10 @@ To start a new Jenkins with Blue Ocean installed:
 . Run `docker run -p 8888:8080 jenkinsci/blueocean:latest`
 . Browse to `http://localhost:8888/blue`
 
+To store your Jenkins data on the host simply replace `/your/home` in the following command with a path on your host and run `docker run -p 8080:8080 -p 50000:50000 -v /your/home:/var/jenkins_home jenkins`.
+
+Or to use with a link:docker run --name myjenkins -p 8080:8080 -p 50000:50000 -v /var/jenkins_home jenkins[Docker Volume] run `docker run --name myjenkins -p 8080:8080 -p 50000:50000 -v /var/jenkins_home jenkins`.
```

Comment:
Both these commands seems involved enough to warrant a `[source]` block with some description. 
Also, instead of `/your/home`, how about `/path/on/host`.